Elections_scraper/election_sc.py

```diff
@@ -0,0 +1,158 @@
+import csv
+import requests
+from bs4 import BeautifulSoup as bs
+import pandas as pd
+
+URL = "https://volby.cz/pls/ps2017nss/ps32?xjazyk=CZ&xkraj=14&xnumnuts=8102"
+
+def main():
+    answer = get_answer()
+    parsed = pull_data(answer)
+    table = find_tab(parsed)
+    rows = find_row(table)
+    villages = [villages_info(row) for row in rows]
+    save_csv(villages)
+
+
+def get_answer():
+    return requests.get(URL)
```

Comment:
You should user `response.raise_for_status()` https://requests.readthedocs.io/en/master/user/quickstart/#response-status-codes so you know when something bad happens and you cannot connect.